tomviz/vtkNonOrthoImagePlaneWidget.h

```diff
@@ -275,6 +275,11 @@ class vtkNonOrthoImagePlaneWidget : public vtkPolyDataSourceWidget
   vtkGetObjectMacro(SelectedPlaneProperty, vtkProperty)
 
   // Description:
+  // Set whether to color map scalars or to render them directly.
+  void SetMapScalars(int val);
```

Comment:
Sure.

Note that `clang-format` issues a bogus complaint here because the macro invocation above does not end with a semi-colon. It wants to indents starting at the comment lines.

Reply:
Done.